recipe/meta.yaml

```diff
@@ -88,19 +88,15 @@ outputs:
         - ninja
         - make  # [unix]
         - cmake-no-system
-      run_constrained:
-        # do not co-install with shared-only lib;
-        # zstd-static contains the shared libs already anyway
-        - zstd <0.0a0
+      host:
+        - {{ pin_subpackage('zstd', exact=True) }}
```

Comment:
Why don't you need xz and lz4-c

Reply:
Why should it need that? Also didn't exist before #62...?